### ISSUE TITLE
iocage upgrade incorrectly reports as failed upgrading from 11.2-RELEASE to 11.3-RELEASE

### ISSUE THREAD

**Issue by vendion** [iocage/iocage#992]
Make sure to follow and check these boxes before submitting an issue! Thank you.

- [x] Supply `iocage --version`
  Version	1.1 RELEASE 2019/01
- [x] Supply the commands used, along with any steps to recreate it.
  `# iocage upgrade -r 11.3-RELEASE test`
- [x] Provide the output from the command you issued.
```
To install the downloaded upgrades, run "/tmp/tmplf8aa2n_ install".
Installing updates...
Kernel updates have been installed.  Please reboot and run
"/tmp/tmplf8aa2n_ install" again to finish installing updates.
Installing updates... done.
No updates are available to install.
Run '/tmp/tmplf8aa2n_ fetch' first.
Command: /tmp/tmplf8aa2n_ -b /iocage/jails/test/root -d /iocage/jails/test/root/var/db/freebsd-update/ -f /iocage/jails/test/root/etc/freebsd-update.conf -r 11.3-RELEASE install failed!
# iocage console test
(inside upgraded jail) # freebsd-version
11.3-RELEASE
```
- [x] Supply what you expected the result or output to be
   No error message stating that the upgrade has failed
- [x] Checked that the problem has not already been fixed on `master` if using
 a stable release.
   I don't see any closed tickets regarding this, may have missed one though

Even though when inside the Jail `freebsd-version` reports that it is 11.3-RELEASE, iocage still shows the jail is 11,2-RELEASE 

```
# iocage get release test
11.2-RELEASE-p11
```

**Comment by adam12b1** [comment]
We are seeing the same problem. The upgrade process always dies on the same generic error, when trying to upgrade a single standalone jail with `iocage upgrade -r 11.3-RELEASE [jailname]`:
```
Command: /tmp/tmplf8aa2n_ -b /iocage/jails/test/root -d /iocage/jails/test/root/var/db/freebsd-update/ -f /iocage/jails/test/root/etc/freebsd-update.conf -r 11.3-RELEASE install failed!
```
We're also having a problem when trying to use `iocage upgrade ALL` (see #971 ), but that seems to be totally separate and much more specific (only about the ALL feature).

**Comment by adam12b1** [comment]
Update on this issue: It appears to be specific to the upgrade from 11.2 to 11.3, and does _not_ occur when upgrading all the way from 11.2 to 12.0-RELEASE.

**Comment by grembo** [comment]
@skarekrow This has been fixed in master (will be part of 1.2). Also, the FreeBSD port has a backported patch now [(iocage-1.1_1).](https://svnweb.freebsd.org/ports/head/sysutils/iocage/files/?view=log), which will be part of 2019Q4. So I think this can be closed.

**Comment by skarekrow** [comment]
@grembo Good point! Thanks.
